airflow/providers/google/provider.yaml

```diff
@@ -88,17 +88,13 @@ dependencies:
   - google-cloud-build>=3.0.0
   - google-cloud-compute>=0.1.0,<2.0.0
   - google-cloud-container>=2.2.0,<3.0.0
-  # google-cloud-dataflow-client of version 0.5.5 requires higher versions of
-  # protobuf and proto-plus libraries which can break other dependencies in the current package.
-  - google-cloud-dataflow-client>=0.5.2,<0.5.5
+  - google-cloud-dataflow-client>=0.5.2
   - google-cloud-dataform>=0.2.0
   - google-cloud-datacatalog>=3.0.0
   - google-cloud-dataplex>=0.1.0
   - google-cloud-dataproc>=3.1.0
   - google-cloud-dataproc-metastore>=1.2.0,<2.0.0
-  # google-cloud-dlp of version 3.8.0+ requires higher versions of
-  # protobuf and proto-plus libraries which can break other dependencies in the current package.
-  - google-cloud-dlp>=3.0.0,<3.8.0
+  - google-cloud-dlp>=3.0.0
```

Comment:
Was something changed since https://github.com/apache/airflow/pull/29234#discussion_r1093176100 ?

Reply:
The protobuf dependencies changed apparently in other packages - the fact that the image builds nicely and test pass are enough proof that we can remove the limits :) 

And build-ci-image job nicely shows that new dlp has been used :)


> google-cloud-dlp==3.8.0